galsim/correlatednoise.py

```diff
@@ -1491,17 +1525,21 @@ def __init__(self, variance, rng=None, scale=None, wcs=None, gsparams=None):
             if not isinstance(wcs, galsim.BaseWCS):
                 raise TypeError("wcs must be a BaseWCS instance")
             if not wcs.isUniform():
-                raise ValueError("Cannot provide non-local wcs")
+                raise ValueError("Cannot provide non-uniform wcs")
         elif scale is not None:
             wcs = galsim.PixelScale(scale)
         else:
             wcs = galsim.PixelScale(1.0)
 
-        # Need variance == xvalue(0,0)
-        # Pixel has flux of f/scale^2, so use f = variance * scale^2
-        image_pix = galsim.Pixel(scale=1.0, flux=variance * wcs.pixelArea(), gsparams=gsparams)
-        world_pix = wcs.toWorld(image_pix)
-        cf_object = galsim.AutoConvolve(world_pix)
-        _BaseCorrelatedNoise.__init__(self, rng, cf_object)
+        # Need variance == xvalue(0,0) after autoconvolution
+        # So the Pixel needs to have an amplitude of sigma at (0,0)
+        import math
```

Comment:
I have a question that is probably naive, but I'm curious:

What is the value of importing math and using it for this square root, instead of using the already-imported numpy.sqrt() method?  If you had not imported either of those modules at this point in the code, you would obviously import math since you don't need something that can take the square root of an array here.  Is it that in your experience it is faster to import and use the math.sqrt() method than it is to use numpy.sqrt()?  Or is there some other reason?